subprojects/shipkit/shipkit.gradle

```diff
@@ -0,0 +1,70 @@
+
+apply plugin: 'maven'
+apply plugin: 'idea'
+apply plugin: 'groovy'
+apply plugin: 'checkstyle'
+apply plugin: 'codenarc'
+
+apply plugin: 'com.gradle.plugin-publish'
+
+checkstyle {
```

Comment:
At some point we should have checkstyle/codenarc (for Groovy) rules enabled for all subprojects. We don't have to do it now, though ;)